### ISSUE TITLE
Screenreader doesnt read error messages

### ISSUE THREAD

**Issue by SandisStudi** [liqd/a4-meinberlin#4596]
**URL:** e.g. https://meinberlin-dev.liqd.net/dashboard/modules/burgerinnenhaushalt-3-phasen-10/generate-codes/

**user:** admin
**expected behaviour:** If I already reached the code generating limit of 5 Mio codes and try to generate more, so that there is a visual error message, that this is not possible, the most used screenreaders (JAWS and NVDA) should also tell me this
**behaviour:** NVDA doesnt read the visually shown error message, that the code generation limit is reached and it is not possible to generate more codes 
**important screensize:** -----
**device & browser:** Mac/Windows with Chrome, Firefox, Edge
**Comment/Question:**

**Comment by CarolingerSeilchenspringer** [comment]
as discussed with @fuzzylogic2000 and @Rineee we need to check if this is true for all error messages, then globally change

**Comment by phillimorland** [comment]
NOTE: error messages for jsx should now be read, I so know that I looked into adding `role=alert` to the django errors but failed, @fuzzylogic2000 @Rineee do you know how to add attributes to default form errors? the info alerts should have a role polite so we need to be able to alter them

**Comment by fuzzylogic2000** [comment]
Uh, yes, I'll have a look! Could you point me at the changes for the jsx, so that I can add the same roles for errors, warnings and info alerts?

**Comment by phillimorland** [comment]
@fuzzylogic2000 so I actually messed up with the jsx and just added alert but i'll change that today but as a temporary solution I did just add it to the surround div in a few places, might be easier? https://github.com/liqd/a4-meinberlin/blob/b75ddedf17e0c937ec657f9871140e5954b703d4/meinberlin/apps/budgeting/templates/meinberlin_budgeting/proposal_list.html#L34
also just searched for `role="alert"` and we might want to have a little overhaul there, there quite a few that are not great

**Comment by fuzzylogic2000** [comment]
@phillimorland Yes, adding it to the surrounding div instead of overwriting whatever Django does might be easier. But to fix it once and for all coming alerts, we should probably add it to the alert code. So, it's good enough for now, but we need to better fix it later?

**Comment by phillimorland** [comment]
@fuzzylogic2000 I mean we assign styling depending on the error so we do kind of know that? so maybe there is a way to change the role too?! are you working on this right now?
